docs/pages/deployment/monitoring.rst

```diff
@@ -73,124 +73,157 @@ It's important to enter the correct IP/domain and port where the Nuts node can b
     # TYPE go_memstats_last_gc_time_seconds gauge
-    go_memstats_last_gc_time_seconds 1.5942182098267434e+09
+    go_memstats_last_gc_time_seconds 1.659085950149366e+09
     # HELP go_memstats_lookups_total Total number of pointer lookups.
     # TYPE go_memstats_lookups_total counter
     go_memstats_lookups_total 0
     # HELP go_memstats_mallocs_total Total number of mallocs.
     # TYPE go_memstats_mallocs_total counter
-    go_memstats_mallocs_total 427334
+    go_memstats_mallocs_total 525508
     # HELP go_memstats_mcache_inuse_bytes Number of bytes in use by mcache structures.
     # TYPE go_memstats_mcache_inuse_bytes gauge
-    go_memstats_mcache_inuse_bytes 13888
+    go_memstats_mcache_inuse_bytes 14400
     # HELP go_memstats_mcache_sys_bytes Number of bytes used for mcache structures obtained from system.
     # TYPE go_memstats_mcache_sys_bytes gauge
-    go_memstats_mcache_sys_bytes 16384
+    go_memstats_mcache_sys_bytes 15600
     # HELP go_memstats_mspan_inuse_bytes Number of bytes in use by mspan structures.
     # TYPE go_memstats_mspan_inuse_bytes gauge
-    go_memstats_mspan_inuse_bytes 115736
+    go_memstats_mspan_inuse_bytes 324360
     # HELP go_memstats_mspan_sys_bytes Number of bytes used for mspan structures obtained from system.
     # TYPE go_memstats_mspan_sys_bytes gauge
-    go_memstats_mspan_sys_bytes 229376
+    go_memstats_mspan_sys_bytes 375360
     # HELP go_memstats_next_gc_bytes Number of heap bytes when next garbage collection will take place.
     # TYPE go_memstats_next_gc_bytes gauge
-    go_memstats_next_gc_bytes 1.6785728e+07
+    go_memstats_next_gc_bytes 1.3709824e+07
     # HELP go_memstats_other_sys_bytes Number of bytes used for other system allocations.
     # TYPE go_memstats_other_sys_bytes gauge
-    go_memstats_other_sys_bytes 1.584792e+06
+    go_memstats_other_sys_bytes 2.696019e+06
     # HELP go_memstats_stack_inuse_bytes Number of bytes in use by the stack allocator.
     # TYPE go_memstats_stack_inuse_bytes gauge
-    go_memstats_stack_inuse_bytes 2.424832e+06
+    go_memstats_stack_inuse_bytes 2.195456e+06
     # HELP go_memstats_stack_sys_bytes Number of bytes obtained from system for stack allocator.
     # TYPE go_memstats_stack_sys_bytes gauge
-    go_memstats_stack_sys_bytes 2.424832e+06
+    go_memstats_stack_sys_bytes 2.195456e+06
     # HELP go_memstats_sys_bytes Number of bytes obtained from system.
     # TYPE go_memstats_sys_bytes gauge
-    go_memstats_sys_bytes 7.2810744e+07
+    go_memstats_sys_bytes 3.1278088e+07
     # HELP go_threads Number of OS threads created.
     # TYPE go_threads gauge
-    go_threads 18
+    go_threads 19
+    # HELP http_request_duration_seconds The HTTP request latencies in seconds.
+    # TYPE http_request_duration_seconds histogram
+    http_request_duration_seconds_bucket{code="500",method="GET",url="/internal/didman/v1/did/:did/compoundservice",le="0.005"} 1
```

Comment:
I'm guessing you'll get other status code lines here as well when you do those requests?
Maybe we have to change docs to make this readable?

Reply:
> I'm guessing you'll get other status code lines here as well when you do those requests? Maybe we have to change docs to make this readable?

Yes, and others for other methods and URIs. Maybe remove the example altogether?